cookbook/bundles/best_practices.rst

```diff
@@ -356,31 +343,74 @@ The end user can provide values in any configuration file:
 
         # app/config/config.yml
         parameters:
-            acme_hello.email.from: fabien@example.com
+            acme_blog.author.email: fabien@example.com
 
     .. code-block:: xml
 
         <!-- app/config/config.xml -->
         <parameters>
-            <parameter key="acme_hello.email.from">fabien@example.com</parameter>
+            <parameter key="acme_blog.author.email">fabien@example.com</parameter>
         </parameters>
 
     .. code-block:: php
 
         // app/config/config.php
-        $container->setParameter('acme_hello.email.from', 'fabien@example.com');
+        $container->setParameter('acme_blog.author.email', 'fabien@example.com');
 
 Retrieve the configuration parameters in your code from the container::
 
-    $container->getParameter('acme_hello.email.from');
+    $container->getParameter('acme_blog.author.email');
 
-Even if this mechanism is simple enough, you are highly encouraged to use the
-:doc:`semantic bundle configuration </cookbook/bundles/extension>` instead.
+Even if this mechanism is simple enough, you should consider using the more
+advanced :doc:`semantic bundle configuration </cookbook/bundles/configuration>`.
 
-.. note::
+Versioning
+----------
+
+Bundles must be versioned following the `Semantic Versioning Standard`_.
+
+Services
+--------
+
+If the bundle defines services, they must be prefixed with the bundle alias.
+For example, AcmeBlogBundle services must be prefixed with ``acme_blog``.
 
-    If you are defining services, they should also be prefixed with the bundle
-    alias.
+In addition, services not meant to be used by the application directly, should
+be :ref:`defined as private <container-private-services>`.
+
+.. seealso::
+
+    You can learn much more about service loading in bundles reading this article:
+    :doc:`How to Load Service Configuration inside a Bundle </cookbook/bundles/extension>`.
+
+Composer Metadata
+-----------------
 
-.. _`standards`: http://www.php-fig.org/psr/psr-0/
-.. _`PHPDoc`: https://en.wikipedia.org/wiki/PHPDoc
+The ``composer.json`` file should include at least the following metadata:
+
+* ``name``, which includes the vendor and the short bundle name. If you are
+  releasing the bundle on your own instead of on behalf of a company, use your
+  personal name (e.g. ``johnsmith/blog-bundle``). The bundle short name excludes
+  the vendor name and separates each word with an hyphen. For example:
+  ``AcmeBlogBundle`` is transformed into ``blog-bundle`` and ``AcmeSocialConnectBundle``
+  is transformed into ``social-connect-bundle``;
+* ``description``, a brief explanation of the purpose of the bundle;
+* ``type``, use the ``symfony-bundle`` value;
+* ``license``, ``MIT`` is the preferred license for Symfony bundles, but you
+  can use any other value.
```

Comment:
by the way, using a definition list rather than an unordered list looks more reasonable to me